plugins/dev-scripts/config/webpack.config.js

```diff
@@ -85,6 +85,9 @@ module.exports = (env) => {
       },
       extensions: ['.ts', '.js']
           .filter((ext) => isTypescript || !ext.includes('ts')),
+      fallback: {
+        'util': false,
+      },
```

Comment:
What was the purpose of this change, @BeksOmega?

Reply:
Guess this was causing problems when building for node, cuz this isn't included as a polyfill anymore. https://webpack.js.org/configuration/resolve/#resolvefallback

I was probably getting some error and this was the suggested solution on stack overflow, cuz I don't remember details.